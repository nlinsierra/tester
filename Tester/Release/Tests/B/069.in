2D 9D 4H 9C 7H JS QH 5C JH AS 4C 8S TC 8D JD QD 6C 4S KC
