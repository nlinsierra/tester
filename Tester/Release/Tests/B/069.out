33
2C 3C 7C 8C JC QC AC 3D 4D 5D 6D 7D TD KD AD 2H 3H 5H 6H 8H 9H TH KH AH 2S 3S 5S 6S 7S 9S TS QS KS
